src/interface.ts

```diff
@@ -90,17 +87,44 @@ interface DesktopAgent {
   open(name: string, context?: Context): Promise<void>;
 
   /**
-   * Finds a mapping of Intents and Apps (action metadata) from an intent & context pair 
+   * Find out more information about a particular intent by passing it's name, and optionally its context.
```

Comment:
`<pedantic>`passing its name`</pedantic>`

Reply:
That typo is fixed now, thanks @ColinEberhardt. It's and its always trip me up...